app/app.go

```diff
@@ -734,7 +734,7 @@ func (app *App) initModules(ctx sdk.Context) {
 
 func (app *App) initCrossChain() {
 	app.CrossChainKeeper.SetSrcChainID(sdk.ChainID(app.appConfig.CrossChain.SrcChainId))
-	app.CrossChainKeeper.SetDestChainID(sdk.ChainID(app.appConfig.CrossChain.DestBscChainId))
+	app.CrossChainKeeper.SetDestBscChainID(sdk.ChainID(app.appConfig.CrossChain.DestBscChainId))
```

Comment:
is DestOpChainID needed?

Reply:
we need a hardfork to register the `DestOpChainID `